src/main/java/net/minestom/server/listener/BlockPlacementListener.java

```diff
@@ -28,6 +28,7 @@
 
 public class BlockPlacementListener {
     private static final BlockManager BLOCK_MANAGER = MinecraftServer.getBlockManager();
+    private static final double BLOCK_HEIGHT = 320.0;
```

Comment:
This isn't constant for every instance, use `instance.getDimensionType().getMaxY()`